tools/stress-cluster/chaos/README.md

```diff
@@ -234,6 +235,57 @@ output RESOURCE_GROUP string = resourceGroup().name
 output APP_CONFIG_NAME string = config.name
 ```
 
+#### Deploying to a Custom Subscription
+
+By default, the stress test environment will load credentials targeting the subscription that the stress cluster is deployed to.
+However it is possible to have tests deploy Azure resources to a custom subscription. This can be useful in cases where resources require ARM feature flags
+that do not exist in the cluster subscription, or if azure resource deployments should be billed to a different team owning the custom subscription.
+
+To set up a custom subscription:
+
+1. Create a service principal with Contributor access to your subscription (or Owner if your bicep file needs to set RBAC policies).
+2. Set a secret with your desired name in the static keyvault provisioned for the stress cluster. The keyvault name can be found in the [addons values config](/tools/stress-cluster/cluster/kubernetes/stress-test-addons/values.yaml) for the desired environment under the key `staticTestSecretsKeyvaultName`.
+
+The secret contents should look like:
+
+```
+AZURE_CLIENT_SECRET=<Service Principal password>
+AZURE_CLIENT_ID=<Service Principal app id>
+AZURE_CLIENT_OID=<Service Principal object id>
+AZURE_TENANT_ID=<AAD tenant ID>
+AZURE_SUBSCRIPTION_ID=<Subscription ID>
+```
+
+3. Update your scenarios-matrix.yaml file to set the `subscriptionConfig` field for the scenarios that should deploy to the custom subscription.
+   The value should match the secret name in keyvault. See [scenarios-matrix.yaml](examples/stress-deployment-example/scenarios-matrix.yaml) or below for examples.
+
+Override subscription for ALL scenarios
+```
+matrix:
+  subscriptionConfig: yourCustomSubscriptionConfigSecret
```

Comment:
maybe keep the name consistent? `staticTestSecretsKeyvaultName`

Reply:
This points to a keyvault secret, not a keyvault resource, so I guess that would be `staticTestKeyvaultSecretName`. However that's a value that I don't think gives much context to a doc reader, so I wanted to have an example name that was a bit more clear about what the value should be replaced with.

Reply:
Perhaps?

```
  subscriptionConfig: <your subscription config secret name>
```